components/compliance-service/api/tests/07_suggestions_spec.rb

```diff
@@ -168,8 +168,6 @@ def reporting
       "apache_extras::harden",
       "apache_extras::windows_harden",
       "apache_extras",
-      "nagios::fix",
-      "java::default",
```

Comment:
I am not sure why these would be suggestions for the term "apache_extras::harden"